server/src/main/java/org/candlepin/model/dto/ContentData.java

```diff
@@ -111,6 +111,38 @@ public ContentData() {
     }
 
     /**
+     * Initializes a new ContentData instance with the specified values.
+     * <p/></p>
+     * <strong>Note</strong>: This constructor passes the provided values to their respective
+     * mutator methods, and does not capture any exceptions they may throw due to malformed
+     * values.
+     *
+     * @param id
+     *  The ID of the content to be represented by this DTO; cannot be null
+     *
+     * @param name
+     *  The name of the content to be represented by this DTO
+     *
+     * @param type
+     *  The type of the content to be represented by this DTO
+     *
+     * @param label
+     *  The label of the content to be represented by this DTO
+     *
+     * @param vendor
+     *  The vendor of the content to be represented by this DTO
+     */
+    public ContentData(String id, String name, String type, String label, String vendor) {
```

Comment:
Keep in mind the constructor in Content that starts with name isn't new -- that's legacy that could be removed or adjusted accordingly. If we're keeping it around, the argument order specified by the method called out here would be my preference (id, name, etc....), as it goes from most to least importance.